.env

```diff
@@ -9,6 +9,7 @@ SENTRY_IMAGE=getsentry/sentry:nightly
 SNUBA_IMAGE=getsentry/snuba:nightly
 RELAY_IMAGE=getsentry/relay:nightly
 SYMBOLICATOR_IMAGE=getsentry/symbolicator:nightly
+
```

Comment:
?

Reply:
Oops, this is a remnant from an earlier iteration of my changes where I had the image/platform info in .env, I'll remove this.